images/sds-drbd-controller/pkg/controller/linstor_node.go

```diff
@@ -384,8 +384,12 @@ func GetLinstorNodes(ctx context.Context, lc *lclient.Client) ([]lclient.Node, [
 		return nil, nil, err
 	}
 
-	linstorControllerNodes := make([]lclient.Node, len(linstorNodes))
-	linstorSatelliteNodes := make([]lclient.Node, len(linstorNodes))
+	linstorControllerNodes := make([]lclient.Node, 0, len(linstorNodes))
+	linstorSatelliteNodes := make([]lclient.Node, 0, len(linstorNodes))
+
+	//TODO: 1
+	// TODO: 2
+	// TODO: 3
```

Comment:
I think todo 3 is not necessary
```suggestion
	linstorControllerNodes := make([]lclient.Node, 0, len(linstorNodes))
	linstorSatelliteNodes := make([]lclient.Node, 0, len(linstorNodes))

	//TODO: 1
	// TODO: 2
	
```